spec/views/events/edit.html.erb_spec.rb

```diff
@@ -7,7 +7,7 @@
     render
 
     assert_select 'form[action=?][method=?]', event_path(event), 'post' do
-      assert_select 'input#event_hashtag[name=?]', 'event[hash_tag]'
+      assert_select 'input#event_hash_tag[name=?]', 'event[hash_tag]'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.